scripts/legislation/africa/eritrea_english.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^CHAPTER I/,$p'
+}
+
+function add_square_brackets_to_list_numbers {
+    sed -E 's/(\()([0-9]+)(\))/[\2]/g' 
```

Comment:
You don't use the parentheses in your replacement text, so you only need to capture the number. As such, this can read:

```
sed -E 's/\(([0-9]+)\)/[\1]/g'
```

Also, this line is indented too much.